article/test/DCRFake.scala

```diff
@@ -8,10 +8,15 @@ import play.api.libs.ws.WSClient
 import play.api.mvc.{RequestHeader, Result}
 import play.twirl.api.Html
 
+import scala.collection.mutable
+import scala.collection.mutable.Queue
 import scala.concurrent.{ExecutionContext, Future}
 
 // It is always a mistake to rely on actual DCR output for tests.
 class DCRFake(implicit context: ApplicationContext) extends renderers.DotcomRenderingService {
+
+  val requestedBlogs: scala.collection.mutable.Queue[PageWithStoryPackage] = new Queue[PageWithStoryPackage]()
```

Comment:
think the type is pretty clear so maybe this reads a bit easier
```suggestion
  val requestedBlogs = new Queue[PageWithStoryPackage]()
```